styles/sass/partials/_drinkList.scss

```diff
@@ -0,0 +1,26 @@
+.drink__list {
+  display: flex;
+  flex-wrap: wrap;
+  align-items: center;
```

Comment:
The list is currently hugging the left side of the window. I would suggest trying `justify-content: center` and maybe adding a wrapper.